### PR TITLE
Clarifications

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

**Comment by dericed** [comment]
This PR is fairly complex and touches on a huge amount of the document. I'm going to wait on some of the next issues assigned to me until this PR is reviewed and merged.

File: specification.markdown [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 
 EBML uses a simple approach of building Elements upon three pieces of data (tag, length, and value) as this approach is well known, easy to parse, and allows selective data parsing. The EBML structure additionally allows for hierarchical arrangement to support complex structural formats in an efficient manner.
 
-## Notation and Conventions
+# Notation and Conventions
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!RFC2119](https://tools.ietf.org/html/rfc2119).
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!RFC2119].
```
Comment (ablwr) [inline specification.markdown]: No more link?

Comment (dericed) [inline specification.markdown]: Link is added in reference section by xml2rfc and mmark when make is run.

File: specification.markdown [hunk 2]
```diff
@@ -304,32 +323,26 @@ When a float value is represented textually in an EBML Schema, such as within a
...
 
-#### Note on the Use of default attributes to define Mandatory EBML Elements
+### Note on the Use of default attributes to define Mandatory EBML Elements
 
-If a Mandatory EBML Element has a default value declared by an EBML Schema and the EBML Element's value is equal to the declared default value then that Element is not required to be present within the EBML Document if its Parent EBML Element is present. In this case, the default value of the Mandatory EBML Element may be assumed although the EBML Element is not present within its Parent EBML Element. Also in this case the parser of the EBML Document MUST insert the defined default value of the Element.
+If a Mandatory EBML Element has a default value declared by an EBML Schema and the EBML Element's value is equal to the declared default value then that Element is not required to be present within the EBML Document if its Parent EBML Element is present. In this case, the default value of the Mandatory EBML Element MUST be interpretted by the EBML Reader although the EBML Element is not present within its Parent EBML Element.
```
Comment (ablwr) [inline specification.markdown]: "interpretted" should be "interpreted"

**Comment by ablwr** [comment]
Other than one misspelling. LGTM. 👍

**Comment by dericed** [comment]
update per @kit90's comment with https://github.com/Matroska-Org/ebml-specification/commit/a5249e379941d6268b00b1f15f591d4f840161e9.

**Comment by dericed** [comment]
fix a type found by @ablwr in https://github.com/Matroska-Org/ebml-specification/commit/7414693411e2581655ecde7bbdd43f041a6d1cfc.

**Comment by dericed** [comment]
all comments thus far are responded to or resolved, please re-review

**Comment by ablwr** [comment]
+1 for merge

**Comment by kit90** [comment]
+1 for merging.

**Comment by retokromer** [comment]
+1 for merging. We can refine later.

**Comment by dericed** [comment]
nudge to @robUx4 @mbunkus, then

**Comment by mbunkus** [comment]
Done. Sorry for making myself so rare over the last couple of days; it'll probably continue until early next week due to too much other activities that require my attention.

**Pull request merged**